Warning in tensor classes

This is new:
```

/u/bangerth/p/deal.II/1/dealii/source/fe/fe_values.cc:1182:17:   instantiated from ‘void dealii::FEValuesViews::internal::do_function_values(const dealii::Vector<Number>&, const dealii::Table<2, double>&, const std::vector<typename dealii::FEValuesViews::Tensor<2, dim, spacedim>::ShapeFunctionData>&, std::vector<typename dealii::ProductType<Number, dealii::Tensor<2, dim> >::type>&) [with int dim = 1, int spacedim = 2, Number = long double, typename dealii::FEValuesViews::Tensor<2, dim, spacedim>::ShapeFunctionData = dealii::FEValuesViews::Tensor<2, 1, 2>::ShapeFunctionData, typename dealii::ProductType<Number, dealii::Tensor<2, dim> >::type = dealii::Tensor<2, 2, long double>]’
/u/bangerth/p/deal.II/1/dealii/source/fe/fe_values.cc:1643:5:   instantiated from ‘void dealii::FEValuesViews::Tensor<2, dim, spacedim>::get_function_values(const InputVector&, std::vector<typename dealii::ProductType<dealii::Tensor<2, spacedim>, typename InputVector::value_type>::type>&) const [with InputVector = dealii::Vector<long double>, int dim = 1, int spacedim = 2, typename dealii::ProductType<dealii::Tensor<2, spacedim>, typename InputVector::value_type>::type = dealii::Tensor<2, 2, long double>]’
source/fe/fe_values.inst:1947:140:   instantiated from here
/u/bangerth/p/deal.II/1/dealii/include/deal.II/base/tensor_accessors.h:565:10: warning: unused parameter ‘indices’ [-Wunused-parameter]
```

--- a/include/deal.II/base/tensor_accessors.h
+++ b/include/deal.II/base/tensor_accessors.h
@@ -545,9 +545,11 @@ namespace TensorAccessors
     struct ExtractHelper
     {
       template<typename T, typename ArrayType>
-      inline static
+      inline
+      static
       typename ReturnType<rank - position, T>::value_type &
-      extract(T &t, const ArrayType &indices)
+      extract(T &t,
+              const ArrayType &indices)
       {
         return ExtractHelper<position + 1, rank>::
                template extract<typename ValueType<T>::value_type, ArrayType>
@@ -561,8 +563,10 @@ namespace TensorAccessors
     struct ExtractHelper<rank, rank>
     {
       template<typename T, typename ArrayType>
-      inline static
-      T &extract(T &t, const ArrayType &indices)
+      inline
+      static
+      T &extract(T &t,
+                 const ArrayType &)
       {
         return t;
       }

--- a/include/deal.II/base/tensor_accessors.h
+++ b/include/deal.II/base/tensor_accessors.h
@@ -557,7 +557,7 @@ namespace TensorAccessors
       }
     };
 
-    // For dimension == rank there is nothing to extract, just return the
+    // For position == rank there is nothing to extract, just return the
     // object.
     template<int rank>
     struct ExtractHelper<rank, rank>

